released/packages/coq-mathcomp-sum-of-two-square/coq-mathcomp-sum-of-two-square.1.0.0/opam

```diff
@@ -6,19 +6,17 @@ license: "MIT"
 build: [
   ["./configure.sh"]
   [make "-j%{jobs}%"]
-  [make "install"]
-]
-remove: [
-  ["rm" "-R" "%{lib}%/coq/user-contrib/mathcomp/contrib/sum_of_two_square"]
-  ["sh" "-c" "rmdir %{lib}%/coq/user-contrib/mathcomp/contrib || true"]
 ]
+install: [make "install"]
 depends: [
   "ocaml"
   "coq" {>= "8.5"}
-  "coq-math-comp" {>= "1.6" & < "1.7"} |
-  ("coq-mathcomp-ssreflect" {>= "1.6" & < "1.7"} &
-   ("coq-mathcomp-algebra" {>= "1.6" & < "1.7"} &
-    "coq-mathcomp-field" {>= "1.6" & < "1.7"}))
+  "coq-mathcomp-ssreflect" {>= "1.6" & < "1.7"} &
```

Comment:
```suggestion
  "coq-mathcomp-ssreflect" {>= "1.6" & < "1.7"}
```
Not sure what this `&` was supposed to be good for, but let's take the chance to remove it.